assets/pages/new-contributions-landing/components/ContributionType.jsx

```diff
@@ -19,16 +20,31 @@ type PropTypes = {
 };
 
 const mapStateToProps = (state: State) => ({
+  state,
   contributionType: state.page.form.contributionType,
+  countryId: state.common.internationalisation.countryId,
+  switches: state.common.settings.switches,
 });
 
 const mapDispatchToProps = (dispatch: Dispatch<Action>) => ({
-  onSelectContributionType: (e) => {
+  dispatch,
+});
+
+function mergeProps(stateProps, dispatchProps, ownProps) {
+
+  const onSelectContributionType = (e) => {
     if (e.target.value !== 'ONE_OFF' && e.target.value !== 'MONTHLY' && e.target.value !== 'ANNUAL') { return; }
+    const paymentMethodToSelect =
+      getValidPaymentMethods(e.target.value, stateProps.switches, stateProps.countryId)[0] || 'None';
```

Comment:
this logic around selecting the first element or `'None'` is now in two places: https://github.com/guardian/support-frontend/blob/jr-fix-tab/assets/pages/new-contributions-landing/contributionsLandingInit.js#L23-L26

I feel like we could extract it out into a function